### PR TITLE
Upgrade to the latest version of Cypress

### LINKED ISSUE #5905: Remove npm audit whitelisted package - minimist
**Describe the bug**
Similarly to https://github.com/bbc/simorgh/issues/5814, we added `minimist` to the `npm audit` whitelist, since there are several packages that need to be updated.

**Top Level Packages to be updated**
- @bbc/moment-timezone-include - https://github.com/bbc/psammead/pull/3266
- @loadable/webpack-plugin
- @storybook/react
- babel-jest
- babel-loader
- bbc-a11y
- brotli-webpack-plugin
- chrome-launcher
- compression-webpack-plugin
- copy-webpack-plugin
- cypress
- eslint
- jest-junit
- jest
- mocha-junit-reporter
- mocha
- storybook-chromatic
- stylelint
- terser-webpack-plugin
- webpack-bundle-analyzer
- webpack-dev-server
- webpack

Full list [here](https://gist.github.com/jblaut/ccee75e6c1e3c8af43467060aa806616)

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves https://github.com/bbc/simorgh-infrastructure/issues/837

**Overall change:** 
_Upgrade to the latest version of Cypress_

**Code changes:**
- _Convert the string timestamp into a number to meet the parameter requirements for [Chai](https://www.chaijs.com/api/bdd/#method_within)._
- _Bump Cypress from 3.8.1 to 4.4.0_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const assertCookieExpiryDate = (cookieName, timestamp) => {
   cy.getCookie(cookieName).then(c => {
     expect(c.expiry).to.be.within(
       timestamp - testBuffer,
-      timestamp + testBuffer,
+      Number(timestamp + testBuffer),
```
Comment (EinsteinNjoroge) [inline cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js]: @FK78 I think `parseInt()` has better error handling than `Number()`.

**Review by j-pendlebury — APPROVED** [review]

**Review by Bopchy — APPROVED** [review]

**Comment by staylos92** [comment]
As long as the current Cypress tests still work with this change then there's no need for a tester to look at this. Moving to ready for merge.

**Comment by FK78** [comment]
> As long as the current Cypress tests still work with this change then there's no need for a tester to look at this. Moving to ready for merge.

24s tests are failing for a single service, `tigrinya`. I am currently looking into what is causing this.

**Comment by amywalkerdev** [comment]
Moving this back to in progress until we verify that the tests pass

**Comment by jamesdonoh** [comment]
@jblaut @FK78 did either of you get any further with this? Let me know if you need a hand, would love to get Cypress updated as soon as possible.

**Comment by j-pendlebury** [comment]
@jamesdonoh I'm still looking into it. Just need to finish some of my write up, but on average tests run slower with the new version of Cypress, possibly because of the new version of Electron. Will update with more info shortly

**Comment by FK78** [comment]
> @jblaut @FK78 did either of you get any further with this? Let me know if you need a hand, would love to get Cypress updated as soon as possible.

Not currently looking into the issue as I am waiting on space to become available on CodeBuild, but last time I ran the job we had 24 tests fail for a single service only, test time did not change significantly.

**Comment by j-pendlebury** [comment]
I'm still looking into this in between other things but here's what I got so far.

I ran the front page e2e's on version 3.8.1 (current) and 4.3.0 (latest version available), with `CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive` and here are the results:

## 3.8.1
Run 1: 3413s => 57min
Run 2: 3477s => 58min
Run 3: 3449s => 57min

**Average: (3413 + 3477 + 3449)/3 = 3446.333333333 => 3446s => 57min**

## 4.3.0
Run 1: 4217s => 1 hour 10 min
Run 2: 4223s => 1 hour 10 min
Run 3: 4368s => 1 hours 13 min

**Average: (4217 + 4223 + 4368)/3 = 4269.333333333 => 4269s => 1 hour 11min**

So difference of about 14 minutes for front page tests.

This was all done locally, so tests ran slower than on CodeBuild. 

Currently the front page tests take about 25 minutes (on TEST, not LIVE) on CodeBuild, so with version 4.3.0, front page tests would take roughly 32 minutes.

Here are some predictions for how long the tests will take with Cypress@4.3.0 on CodeBuild based on how long they run for now:
image
Skipped tests which take seconds or milliseconds to run (e.g. errorPage tests)

Also found these two issues:
- https://github.com/cypress-io/cypress/issues/5824
- https://github.com/cypress-io/cypress/issues/6388

which shows we're not the only ones experiencing this issue

**Comment by amywalkerdev** [comment]
Thanks for the analysis Jamie, I'm happy to update cypress knowing it will add time to our runs for the moment, then we can look to streamline and improve our tests after.

**Comment by jamesdonoh** [comment]
+1 thanks Jamie and Fahad. I think we should push ahead with this too.

**Comment by FK78** [comment]
> +1 thanks Jamie and Fahad. I think we should push ahead with this too.

A newer version of Cypress is available, I am going to run it on CodeBuild before we go a

**Pull request merged**